CRM/SMS/Form/Upload.php

```diff
@@ -44,7 +47,7 @@ public function setDefaultValues() {
     $this->assign('count', $count);
 
     $this->set('skipTextFile', FALSE);
-
+    $skipTextFile = FALSE;
```

Comment:
Minor note: You wouldn't need the set() just above anymore.

Otherwise this is a bit hard to test because clicking continue on an existing draft creates a new one (which seems like a different bug than your other PR?), so I don't think things are displaying properly. But having said that I don't see anything worse happening than before.

Reply:
ah I didn't see that ok will pull these parts off and yeh didn't notice the draft issue when i was testing i don't think